src/scss/x.scss

```diff
@@ -0,0 +1,36 @@
+.x {
+  display: block;
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Woah there buddy, a bit trigger happy perhaps? I suggest you remove the many empty lines

Reply:
This is fixed in ffa004083d114d9a860e1da51f0c89a1bb7a9c37